modules/cluster-logging-collector-log-forward-about.adoc

```diff
@@ -5,27 +5,27 @@
 [id="cluster-logging-collector-log-forward-about_{context}"]
 = Understanding the Log Forwarding API
 
-Forwarding cluster logs using the Log Forwarding API requires a combination of _outputs_ and _pipelines_ to send logs to specific endpoints inside and outside of your {product-title} cluster. 
+Forwarding cluster logs using the Log Forwarding API requires a combination of _outputs_ and _pipelines_ to send logs to specific endpoints inside and outside of your {product-title} cluster.
```

Comment:
Kind of a wordy sentence. Could restructure if you feel like it.

Also, ISG typically avoids the ambiguity of "using" by subbing it with "by using" or "that uses," depending on context.